packages/code-du-travail-frontend/pages/information/[slug].tsx

```diff
@@ -113,19 +118,36 @@ const Information = ({
       );
     }
   );
-  if (editorialContent.length > 1) {
-    editorialContent = (
-      <Accordion
-        titleLevel={2}
-        preExpanded={[anchor]}
-        items={contents.map(({ title, name }, index) => ({
-          body: editorialContent[index],
-          id: name,
-          title,
-        }))}
-      />
-    );
+  let contentWrapper;
+  if (editorialContent && editorialContent.length > 1) {
+    contentWrapper =
+      sectionDisplayMode === "tab" ? (
+        <Tabs
+          data={contents?.map(({ title, name }, index) => ({
+            panel: editorialContent?.[index],
+            tab: title,
+          }))}
+        />
+      ) : (
+        <Accordion
+          titleLevel={2}
+          preExpanded={[anchor]}
+          items={contents?.map(({ title, name }, index) => ({
+            body: editorialContent?.[index],
+            id: name,
+            title,
+          }))}
+        />
+      );
   }
+  let sectionTitleStyleWrapper =
+    sectionDisplayMode === "tab" ? (
+      <TabStylesWrapper data-testid="tabs">{contentWrapper}</TabStylesWrapper>
+    ) : (
+      <GlobalStylesWrapper data-testid="accordion">
```

Comment:
J'avoue que le `data-testid`, simplifie vraiment la vie. Perso, je pense, plus en rajouter sur mes composants pour les retrouver plus facilement.

Vous en pensez quoi ?

Reply:
Un bon article sur le sujet : https://betterprogramming.pub/why-you-should-avoid-testing-react-components-with-test-ids-ee50d20d37d2

Je le rejoins, je ne suis pas contre les data-testid mais si nécessaire et pas par défaut.